[2020-01-02 16:00:35] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'App\Register' not found in C:\LoginsApi\app\Http\Controllers\RegisterController.php:30
Stack trace:
#0 [internal function]: App\Http\Controllers\RegisterController->index()
#1 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#4 C:\LoginsApi\vendor\illuminate\container\Container.php(572): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#5 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(378): Illuminate\Container\Container->call(Array, Array)
#6 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(344): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#7 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(318): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\RegisterController), 'index', Array)
#8 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(280): Laravel\Lumen\Application->callControllerAction(Array)
#9 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(265): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(418): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#12 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#13 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#14 C:\LoginsApi\public\index.php(28): Laravel\Lumen\Application->run()
#15 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Class 'App\\Register' not found at C:\\LoginsApi\\app\\Http\\Controllers\\RegisterController.php:30)
[stacktrace]
#0 [internal function]: App\\Http\\Controllers\\RegisterController->index()
#1 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#3 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#4 C:\\LoginsApi\\vendor\\illuminate\\container\\Container.php(572): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#5 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(378): Illuminate\\Container\\Container->call(Array, Array)
#6 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(344): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#7 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(318): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\RegisterController), 'index', Array)
#8 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(280): Laravel\\Lumen\\Application->callControllerAction(Array)
#9 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(265): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#10 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#11 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(418): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#12 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#13 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#14 C:\\LoginsApi\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#15 {main}
"} 
[2020-01-02 16:32:33] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'App\Http\Controllers\Login' not found in C:\LoginsApi\app\Http\Controllers\RegisterController.php:102
Stack trace:
#0 [internal function]: App\Http\Controllers\RegisterController->destroy('1')
#1 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#4 C:\LoginsApi\vendor\illuminate\container\Container.php(572): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#5 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(378): Illuminate\Container\Container->call(Array, Array)
#6 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(344): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#7 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(318): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\RegisterController), 'destroy', Array)
#8 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(280): Laravel\Lumen\Application->callControllerAction(Array)
#9 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(265): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(235): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(169): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(418): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\LoginsApi\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Class 'App\\Http\\Controllers\\Login' not found at C:\\LoginsApi\\app\\Http\\Controllers\\RegisterController.php:102)
[stacktrace]
#0 [internal function]: App\\Http\\Controllers\\RegisterController->destroy('1')
#1 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#3 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#4 C:\\LoginsApi\\vendor\\illuminate\\container\\Container.php(572): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#5 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(378): Illuminate\\Container\\Container->call(Array, Array)
#6 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(344): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#7 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(318): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\RegisterController), 'destroy', Array)
#8 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(280): Laravel\\Lumen\\Application->callControllerAction(Array)
#9 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(265): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#10 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(235): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#11 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(169): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#12 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(418): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\LoginsApi\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
[2020-01-02 16:36:19] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'App\Http\Controllers\Login' not found in C:\LoginsApi\app\Http\Controllers\RegisterController.php:61
Stack trace:
#0 [internal function]: App\Http\Controllers\RegisterController->show('1')
#1 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#4 C:\LoginsApi\vendor\illuminate\container\Container.php(572): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#5 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(378): Illuminate\Container\Container->call(Array, Array)
#6 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(344): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#7 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(318): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\RegisterController), 'show', Array)
#8 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(280): Laravel\Lumen\Application->callControllerAction(Array)
#9 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(265): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(235): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(169): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(418): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\LoginsApi\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Class 'App\\Http\\Controllers\\Login' not found at C:\\LoginsApi\\app\\Http\\Controllers\\RegisterController.php:61)
[stacktrace]
#0 [internal function]: App\\Http\\Controllers\\RegisterController->show('1')
#1 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#3 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#4 C:\\LoginsApi\\vendor\\illuminate\\container\\Container.php(572): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#5 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(378): Illuminate\\Container\\Container->call(Array, Array)
#6 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(344): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#7 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(318): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\RegisterController), 'show', Array)
#8 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(280): Laravel\\Lumen\\Application->callControllerAction(Array)
#9 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(265): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#10 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(235): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#11 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(169): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#12 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(418): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\LoginsApi\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
[2020-01-02 16:36:34] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'App\Http\Controllers\Login' not found in C:\LoginsApi\app\Http\Controllers\RegisterController.php:61
Stack trace:
#0 [internal function]: App\Http\Controllers\RegisterController->show('1')
#1 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#4 C:\LoginsApi\vendor\illuminate\container\Container.php(572): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#5 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(378): Illuminate\Container\Container->call(Array, Array)
#6 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(344): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#7 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(318): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\RegisterController), 'show', Array)
#8 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(280): Laravel\Lumen\Application->callControllerAction(Array)
#9 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(265): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(235): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(169): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(418): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\LoginsApi\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Class 'App\\Http\\Controllers\\Login' not found at C:\\LoginsApi\\app\\Http\\Controllers\\RegisterController.php:61)
[stacktrace]
#0 [internal function]: App\\Http\\Controllers\\RegisterController->show('1')
#1 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#3 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#4 C:\\LoginsApi\\vendor\\illuminate\\container\\Container.php(572): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#5 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(378): Illuminate\\Container\\Container->call(Array, Array)
#6 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(344): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#7 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(318): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\RegisterController), 'show', Array)
#8 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(280): Laravel\\Lumen\\Application->callControllerAction(Array)
#9 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(265): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#10 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(235): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#11 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(169): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#12 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(418): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\LoginsApi\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
[2020-01-02 16:37:06] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'App\Http\Controllers\Login' not found in C:\LoginsApi\app\Http\Controllers\RegisterController.php:61
Stack trace:
#0 [internal function]: App\Http\Controllers\RegisterController->show('2')
#1 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#4 C:\LoginsApi\vendor\illuminate\container\Container.php(572): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#5 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(378): Illuminate\Container\Container->call(Array, Array)
#6 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(344): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#7 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(318): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\RegisterController), 'show', Array)
#8 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(280): Laravel\Lumen\Application->callControllerAction(Array)
#9 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(265): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(235): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(169): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(418): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\LoginsApi\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Class 'App\\Http\\Controllers\\Login' not found at C:\\LoginsApi\\app\\Http\\Controllers\\RegisterController.php:61)
[stacktrace]
#0 [internal function]: App\\Http\\Controllers\\RegisterController->show('2')
#1 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#3 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#4 C:\\LoginsApi\\vendor\\illuminate\\container\\Container.php(572): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#5 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(378): Illuminate\\Container\\Container->call(Array, Array)
#6 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(344): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#7 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(318): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\RegisterController), 'show', Array)
#8 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(280): Laravel\\Lumen\\Application->callControllerAction(Array)
#9 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(265): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#10 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(235): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#11 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(169): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#12 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(418): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\LoginsApi\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
[2020-01-02 16:37:27] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'App\Http\Controllers\Login' not found in C:\LoginsApi\app\Http\Controllers\RegisterController.php:61
Stack trace:
#0 [internal function]: App\Http\Controllers\RegisterController->show('1')
#1 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#4 C:\LoginsApi\vendor\illuminate\container\Container.php(572): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#5 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(378): Illuminate\Container\Container->call(Array, Array)
#6 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(344): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#7 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(318): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\RegisterController), 'show', Array)
#8 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(280): Laravel\Lumen\Application->callControllerAction(Array)
#9 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(265): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(235): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(169): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(418): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\LoginsApi\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Class 'App\\Http\\Controllers\\Login' not found at C:\\LoginsApi\\app\\Http\\Controllers\\RegisterController.php:61)
[stacktrace]
#0 [internal function]: App\\Http\\Controllers\\RegisterController->show('1')
#1 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#3 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#4 C:\\LoginsApi\\vendor\\illuminate\\container\\Container.php(572): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#5 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(378): Illuminate\\Container\\Container->call(Array, Array)
#6 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(344): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#7 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(318): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\RegisterController), 'show', Array)
#8 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(280): Laravel\\Lumen\\Application->callControllerAction(Array)
#9 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(265): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#10 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(235): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#11 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(169): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#12 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(418): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\LoginsApi\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
[2020-01-02 16:37:57] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'App\Http\Controllers\Login' not found in C:\LoginsApi\app\Http\Controllers\RegisterController.php:61
Stack trace:
#0 [internal function]: App\Http\Controllers\RegisterController->show('1')
#1 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#4 C:\LoginsApi\vendor\illuminate\container\Container.php(572): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#5 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(378): Illuminate\Container\Container->call(Array, Array)
#6 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(344): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#7 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(318): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\RegisterController), 'show', Array)
#8 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(280): Laravel\Lumen\Application->callControllerAction(Array)
#9 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(265): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(235): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(169): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(418): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\LoginsApi\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Class 'App\\Http\\Controllers\\Login' not found at C:\\LoginsApi\\app\\Http\\Controllers\\RegisterController.php:61)
[stacktrace]
#0 [internal function]: App\\Http\\Controllers\\RegisterController->show('1')
#1 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#3 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#4 C:\\LoginsApi\\vendor\\illuminate\\container\\Container.php(572): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#5 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(378): Illuminate\\Container\\Container->call(Array, Array)
#6 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(344): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#7 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(318): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\RegisterController), 'show', Array)
#8 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(280): Laravel\\Lumen\\Application->callControllerAction(Array)
#9 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(265): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#10 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(235): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#11 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(169): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#12 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(418): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\LoginsApi\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
[2020-01-02 16:39:31] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'App\Http\Controllers\Login' not found in C:\LoginsApi\app\Http\Controllers\RegisterController.php:61
Stack trace:
#0 [internal function]: App\Http\Controllers\RegisterController->show('1')
#1 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#4 C:\LoginsApi\vendor\illuminate\container\Container.php(572): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#5 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(378): Illuminate\Container\Container->call(Array, Array)
#6 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(344): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#7 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(318): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\RegisterController), 'show', Array)
#8 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(280): Laravel\Lumen\Application->callControllerAction(Array)
#9 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(265): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(235): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(169): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(418): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\LoginsApi\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Class 'App\\Http\\Controllers\\Login' not found at C:\\LoginsApi\\app\\Http\\Controllers\\RegisterController.php:61)
[stacktrace]
#0 [internal function]: App\\Http\\Controllers\\RegisterController->show('1')
#1 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#3 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#4 C:\\LoginsApi\\vendor\\illuminate\\container\\Container.php(572): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#5 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(378): Illuminate\\Container\\Container->call(Array, Array)
#6 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(344): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#7 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(318): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\RegisterController), 'show', Array)
#8 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(280): Laravel\\Lumen\\Application->callControllerAction(Array)
#9 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(265): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#10 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(235): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#11 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(169): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#12 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(418): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\LoginsApi\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
[2020-01-02 16:40:32] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'App\Http\Controllers\Login' not found in C:\LoginsApi\app\Http\Controllers\RegisterController.php:102
Stack trace:
#0 [internal function]: App\Http\Controllers\RegisterController->destroy('1')
#1 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#4 C:\LoginsApi\vendor\illuminate\container\Container.php(572): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#5 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(378): Illuminate\Container\Container->call(Array, Array)
#6 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(344): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#7 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(318): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\RegisterController), 'destroy', Array)
#8 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(280): Laravel\Lumen\Application->callControllerAction(Array)
#9 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(265): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#10 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(235): Laravel\Lumen\Application->handleFoundRoute(Array)
#11 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(169): Laravel\Lumen\Application->handleDispatcherResponse(Array)
#12 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(418): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\LoginsApi\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Class 'App\\Http\\Controllers\\Login' not found at C:\\LoginsApi\\app\\Http\\Controllers\\RegisterController.php:102)
[stacktrace]
#0 [internal function]: App\\Http\\Controllers\\RegisterController->destroy('1')
#1 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(29): call_user_func_array(Array, Array)
#2 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#3 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#4 C:\\LoginsApi\\vendor\\illuminate\\container\\Container.php(572): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#5 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(378): Illuminate\\Container\\Container->call(Array, Array)
#6 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(344): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#7 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(318): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\RegisterController), 'destroy', Array)
#8 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(280): Laravel\\Lumen\\Application->callControllerAction(Array)
#9 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(265): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#10 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(235): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#11 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(169): Laravel\\Lumen\\Application->handleDispatcherResponse(Array)
#12 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(418): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\LoginsApi\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
[2020-01-02 16:41:59] local.ERROR: PDOException: SQLSTATE[23000]: Integrity constraint violation: 19 UNIQUE constraint failed: registers.email in C:\LoginsApi\vendor\illuminate\database\Connection.php:458
Stack trace:
#0 C:\LoginsApi\vendor\illuminate\database\Connection.php(458): PDOStatement->execute()
#1 C:\LoginsApi\vendor\illuminate\database\Connection.php(657): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into "re...', Array)
#2 C:\LoginsApi\vendor\illuminate\database\Connection.php(624): Illuminate\Database\Connection->runQueryCallback('insert into "re...', Array, Object(Closure))
#3 C:\LoginsApi\vendor\illuminate\database\Connection.php(459): Illuminate\Database\Connection->run('insert into "re...', Array, Object(Closure))
#4 C:\LoginsApi\vendor\illuminate\database\Connection.php(411): Illuminate\Database\Connection->statement('insert into "re...', Array)
#5 C:\LoginsApi\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into "re...', Array)
#6 C:\LoginsApi\vendor\illuminate\database\Query\Builder.php(2628): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into "re...', Array, 'id')
#7 C:\LoginsApi\vendor\illuminate\database\Eloquent\Builder.php(1338): Illuminate\Database\Query\Builder->insertGetId(Array, 'id')
#8 C:\LoginsApi\vendor\illuminate\database\Eloquent\Model.php(823): Illuminate\Database\Eloquent\Builder->__call('insertGetId', Array)
#9 C:\LoginsApi\vendor\illuminate\database\Eloquent\Model.php(788): Illuminate\Database\Eloquent\Model->insertAndSetId(Object(Illuminate\Database\Eloquent\Builder), Array)
#10 C:\LoginsApi\vendor\illuminate\database\Eloquent\Model.php(651): Illuminate\Database\Eloquent\Model->performInsert(Object(Illuminate\Database\Eloquent\Builder))
#11 C:\LoginsApi\vendor\illuminate\database\Eloquent\Builder.php(793): Illuminate\Database\Eloquent\Model->save()
#12 C:\LoginsApi\vendor\illuminate\support\helpers.php(1027): Illuminate\Database\Eloquent\Builder->Illuminate\Database\Eloquent\{closure}(Object(App\Register))
#13 C:\LoginsApi\vendor\illuminate\database\Eloquent\Builder.php(794): tap(Object(App\Register), Object(Closure))
#14 C:\LoginsApi\vendor\illuminate\support\Traits\ForwardsCalls.php(23): Illuminate\Database\Eloquent\Builder->create(Array)
#15 C:\LoginsApi\vendor\illuminate\database\Eloquent\Model.php(1610): Illuminate\Database\Eloquent\Model->forwardCallTo(Object(Illuminate\Database\Eloquent\Builder), 'create', Array)
#16 C:\LoginsApi\vendor\illuminate\database\Eloquent\Model.php(1622): Illuminate\Database\Eloquent\Model->__call('create', Array)
#17 C:\LoginsApi\app\Http\Controllers\RegisterController.php(50): Illuminate\Database\Eloquent\Model::__callStatic('create', Array)
#18 [internal function]: App\Http\Controllers\RegisterController->store(Object(Laravel\Lumen\Http\Request))
#19 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(29): call_user_func_array(Array, Array)
#20 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#21 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#22 C:\LoginsApi\vendor\illuminate\container\Container.php(572): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#23 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(378): Illuminate\Container\Container->call(Array, Array)
#24 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(344): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#25 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(318): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\RegisterController), 'store', Array)
#26 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(280): Laravel\Lumen\Application->callControllerAction(Array)
#27 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(265): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#28 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#29 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(418): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#30 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#31 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#32 C:\LoginsApi\public\index.php(28): Laravel\Lumen\Application->run()
#33 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 19 UNIQUE constraint failed: registers.email (SQL: insert into "registers" ("name", "email", "password", "username", "updated_at", "created_at") values (test, test03@test.com, password, username1, 2020-01-02 16:41:59, 2020-01-02 16:41:59)) in C:\LoginsApi\vendor\illuminate\database\Connection.php:664
Stack trace:
#0 C:\LoginsApi\vendor\illuminate\database\Connection.php(624): Illuminate\Database\Connection->runQueryCallback('insert into "re...', Array, Object(Closure))
#1 C:\LoginsApi\vendor\illuminate\database\Connection.php(459): Illuminate\Database\Connection->run('insert into "re...', Array, Object(Closure))
#2 C:\LoginsApi\vendor\illuminate\database\Connection.php(411): Illuminate\Database\Connection->statement('insert into "re...', Array)
#3 C:\LoginsApi\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into "re...', Array)
#4 C:\LoginsApi\vendor\illuminate\database\Query\Builder.php(2628): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into "re...', Array, 'id')
#5 C:\LoginsApi\vendor\illuminate\database\Eloquent\Builder.php(1338): Illuminate\Database\Query\Builder->insertGetId(Array, 'id')
#6 C:\LoginsApi\vendor\illuminate\database\Eloquent\Model.php(823): Illuminate\Database\Eloquent\Builder->__call('insertGetId', Array)
#7 C:\LoginsApi\vendor\illuminate\database\Eloquent\Model.php(788): Illuminate\Database\Eloquent\Model->insertAndSetId(Object(Illuminate\Database\Eloquent\Builder), Array)
#8 C:\LoginsApi\vendor\illuminate\database\Eloquent\Model.php(651): Illuminate\Database\Eloquent\Model->performInsert(Object(Illuminate\Database\Eloquent\Builder))
#9 C:\LoginsApi\vendor\illuminate\database\Eloquent\Builder.php(793): Illuminate\Database\Eloquent\Model->save()
#10 C:\LoginsApi\vendor\illuminate\support\helpers.php(1027): Illuminate\Database\Eloquent\Builder->Illuminate\Database\Eloquent\{closure}(Object(App\Register))
#11 C:\LoginsApi\vendor\illuminate\database\Eloquent\Builder.php(794): tap(Object(App\Register), Object(Closure))
#12 C:\LoginsApi\vendor\illuminate\support\Traits\ForwardsCalls.php(23): Illuminate\Database\Eloquent\Builder->create(Array)
#13 C:\LoginsApi\vendor\illuminate\database\Eloquent\Model.php(1610): Illuminate\Database\Eloquent\Model->forwardCallTo(Object(Illuminate\Database\Eloquent\Builder), 'create', Array)
#14 C:\LoginsApi\vendor\illuminate\database\Eloquent\Model.php(1622): Illuminate\Database\Eloquent\Model->__call('create', Array)
#15 C:\LoginsApi\app\Http\Controllers\RegisterController.php(50): Illuminate\Database\Eloquent\Model::__callStatic('create', Array)
#16 [internal function]: App\Http\Controllers\RegisterController->store(Object(Laravel\Lumen\Http\Request))
#17 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(29): call_user_func_array(Array, Array)
#18 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#19 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#20 C:\LoginsApi\vendor\illuminate\container\Container.php(572): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#21 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(378): Illuminate\Container\Container->call(Array, Array)
#22 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(344): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#23 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(318): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\RegisterController), 'store', Array)
#24 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(280): Laravel\Lumen\Application->callControllerAction(Array)
#25 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(265): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#26 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#27 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(418): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#28 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#29 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#30 C:\LoginsApi\public\index.php(28): Laravel\Lumen\Application->run()
#31 {main} {"exception":"[object] (Illuminate\\Database\\QueryException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 19 UNIQUE constraint failed: registers.email (SQL: insert into \"registers\" (\"name\", \"email\", \"password\", \"username\", \"updated_at\", \"created_at\") values (test, test03@test.com, password, username1, 2020-01-02 16:41:59, 2020-01-02 16:41:59)) at C:\\LoginsApi\\vendor\\illuminate\\database\\Connection.php:664, PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 19 UNIQUE constraint failed: registers.email at C:\\LoginsApi\\vendor\\illuminate\\database\\Connection.php:458)
[stacktrace]
#0 C:\\LoginsApi\\vendor\\illuminate\\database\\Connection.php(624): Illuminate\\Database\\Connection->runQueryCallback('insert into \"re...', Array, Object(Closure))
#1 C:\\LoginsApi\\vendor\\illuminate\\database\\Connection.php(459): Illuminate\\Database\\Connection->run('insert into \"re...', Array, Object(Closure))
#2 C:\\LoginsApi\\vendor\\illuminate\\database\\Connection.php(411): Illuminate\\Database\\Connection->statement('insert into \"re...', Array)
#3 C:\\LoginsApi\\vendor\\illuminate\\database\\Query\\Processors\\Processor.php(32): Illuminate\\Database\\Connection->insert('insert into \"re...', Array)
#4 C:\\LoginsApi\\vendor\\illuminate\\database\\Query\\Builder.php(2628): Illuminate\\Database\\Query\\Processors\\Processor->processInsertGetId(Object(Illuminate\\Database\\Query\\Builder), 'insert into \"re...', Array, 'id')
#5 C:\\LoginsApi\\vendor\\illuminate\\database\\Eloquent\\Builder.php(1338): Illuminate\\Database\\Query\\Builder->insertGetId(Array, 'id')
#6 C:\\LoginsApi\\vendor\\illuminate\\database\\Eloquent\\Model.php(823): Illuminate\\Database\\Eloquent\\Builder->__call('insertGetId', Array)
#7 C:\\LoginsApi\\vendor\\illuminate\\database\\Eloquent\\Model.php(788): Illuminate\\Database\\Eloquent\\Model->insertAndSetId(Object(Illuminate\\Database\\Eloquent\\Builder), Array)
#8 C:\\LoginsApi\\vendor\\illuminate\\database\\Eloquent\\Model.php(651): Illuminate\\Database\\Eloquent\\Model->performInsert(Object(Illuminate\\Database\\Eloquent\\Builder))
#9 C:\\LoginsApi\\vendor\\illuminate\\database\\Eloquent\\Builder.php(793): Illuminate\\Database\\Eloquent\\Model->save()
#10 C:\\LoginsApi\\vendor\\illuminate\\support\\helpers.php(1027): Illuminate\\Database\\Eloquent\\Builder->Illuminate\\Database\\Eloquent\\{closure}(Object(App\\Register))
#11 C:\\LoginsApi\\vendor\\illuminate\\database\\Eloquent\\Builder.php(794): tap(Object(App\\Register), Object(Closure))
#12 C:\\LoginsApi\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php(23): Illuminate\\Database\\Eloquent\\Builder->create(Array)
#13 C:\\LoginsApi\\vendor\\illuminate\\database\\Eloquent\\Model.php(1610): Illuminate\\Database\\Eloquent\\Model->forwardCallTo(Object(Illuminate\\Database\\Eloquent\\Builder), 'create', Array)
#14 C:\\LoginsApi\\vendor\\illuminate\\database\\Eloquent\\Model.php(1622): Illuminate\\Database\\Eloquent\\Model->__call('create', Array)
#15 C:\\LoginsApi\\app\\Http\\Controllers\\RegisterController.php(50): Illuminate\\Database\\Eloquent\\Model::__callStatic('create', Array)
#16 [internal function]: App\\Http\\Controllers\\RegisterController->store(Object(Laravel\\Lumen\\Http\\Request))
#17 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(29): call_user_func_array(Array, Array)
#18 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#19 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#20 C:\\LoginsApi\\vendor\\illuminate\\container\\Container.php(572): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#21 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(378): Illuminate\\Container\\Container->call(Array, Array)
#22 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(344): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#23 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(318): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\RegisterController), 'store', Array)
#24 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(280): Laravel\\Lumen\\Application->callControllerAction(Array)
#25 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(265): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#26 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#27 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(418): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#28 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#29 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#30 C:\\LoginsApi\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#31 {main}
"} 
[2020-01-02 21:57:21] local.ERROR: InvalidArgumentException: View [logins] not found. in C:\LoginsApi\vendor\illuminate\view\FileViewFinder.php:137
Stack trace:
#0 C:\LoginsApi\vendor\illuminate\view\FileViewFinder.php(79): Illuminate\View\FileViewFinder->findInPaths('logins', Array)
#1 C:\LoginsApi\vendor\illuminate\view\Factory.php(130): Illuminate\View\FileViewFinder->find('logins')
#2 C:\LoginsApi\vendor\laravel\lumen-framework\src\helpers.php(431): Illuminate\View\Factory->make('logins', Array, Array)
#3 C:\LoginsApi\routes\web.php(32): view('logins')
#4 [internal function]: Closure->{closure}()
#5 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(29): call_user_func_array(Object(Closure), Array)
#6 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(75): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#7 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Object(Closure), Object(Closure))
#8 C:\LoginsApi\vendor\illuminate\container\Container.php(572): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Object(Closure), Array, NULL)
#9 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(291): Illuminate\Container\Container->call(Object(Closure), Array)
#10 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(265): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#11 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#12 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(418): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\LoginsApi\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (InvalidArgumentException(code: 0): View [logins] not found. at C:\\LoginsApi\\vendor\\illuminate\\view\\FileViewFinder.php:137)
[stacktrace]
#0 C:\\LoginsApi\\vendor\\illuminate\\view\\FileViewFinder.php(79): Illuminate\\View\\FileViewFinder->findInPaths('logins', Array)
#1 C:\\LoginsApi\\vendor\\illuminate\\view\\Factory.php(130): Illuminate\\View\\FileViewFinder->find('logins')
#2 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\helpers.php(431): Illuminate\\View\\Factory->make('logins', Array, Array)
#3 C:\\LoginsApi\outes\\web.php(32): view('logins')
#4 [internal function]: Closure->{closure}()
#5 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(29): call_user_func_array(Object(Closure), Array)
#6 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(75): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#7 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Object(Closure), Object(Closure))
#8 C:\\LoginsApi\\vendor\\illuminate\\container\\Container.php(572): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Object(Closure), Array, NULL)
#9 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(291): Illuminate\\Container\\Container->call(Object(Closure), Array)
#10 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(265): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#11 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#12 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(418): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\LoginsApi\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
[2020-01-02 21:59:54] local.ERROR: InvalidArgumentException: View [app] not found. in C:\LoginsApi\vendor\illuminate\view\FileViewFinder.php:137
Stack trace:
#0 C:\LoginsApi\vendor\illuminate\view\FileViewFinder.php(79): Illuminate\View\FileViewFinder->findInPaths('app', Array)
#1 C:\LoginsApi\vendor\illuminate\view\Factory.php(130): Illuminate\View\FileViewFinder->find('app')
#2 C:\LoginsApi\vendor\laravel\lumen-framework\src\helpers.php(431): Illuminate\View\Factory->make('app', Array, Array)
#3 C:\LoginsApi\routes\web.php(32): view('app')
#4 [internal function]: Closure->{closure}()
#5 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(29): call_user_func_array(Object(Closure), Array)
#6 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(75): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#7 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Object(Closure), Object(Closure))
#8 C:\LoginsApi\vendor\illuminate\container\Container.php(572): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Object(Closure), Array, NULL)
#9 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(291): Illuminate\Container\Container->call(Object(Closure), Array)
#10 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(265): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#11 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#12 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(418): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\LoginsApi\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (InvalidArgumentException(code: 0): View [app] not found. at C:\\LoginsApi\\vendor\\illuminate\\view\\FileViewFinder.php:137)
[stacktrace]
#0 C:\\LoginsApi\\vendor\\illuminate\\view\\FileViewFinder.php(79): Illuminate\\View\\FileViewFinder->findInPaths('app', Array)
#1 C:\\LoginsApi\\vendor\\illuminate\\view\\Factory.php(130): Illuminate\\View\\FileViewFinder->find('app')
#2 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\helpers.php(431): Illuminate\\View\\Factory->make('app', Array, Array)
#3 C:\\LoginsApi\outes\\web.php(32): view('app')
#4 [internal function]: Closure->{closure}()
#5 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(29): call_user_func_array(Object(Closure), Array)
#6 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(75): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#7 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Object(Closure), Object(Closure))
#8 C:\\LoginsApi\\vendor\\illuminate\\container\\Container.php(572): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Object(Closure), Array, NULL)
#9 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(291): Illuminate\\Container\\Container->call(Object(Closure), Array)
#10 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(265): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#11 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#12 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(418): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\LoginsApi\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
[2020-01-02 22:03:33] local.ERROR: InvalidArgumentException: View [login] not found. in C:\LoginsApi\vendor\illuminate\view\FileViewFinder.php:137
Stack trace:
#0 C:\LoginsApi\vendor\illuminate\view\FileViewFinder.php(79): Illuminate\View\FileViewFinder->findInPaths('login', Array)
#1 C:\LoginsApi\vendor\illuminate\view\Factory.php(130): Illuminate\View\FileViewFinder->find('login')
#2 C:\LoginsApi\vendor\laravel\lumen-framework\src\helpers.php(431): Illuminate\View\Factory->make('login', Array, Array)
#3 C:\LoginsApi\routes\web.php(32): view('login')
#4 [internal function]: Closure->{closure}()
#5 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(29): call_user_func_array(Object(Closure), Array)
#6 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(75): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#7 C:\LoginsApi\vendor\illuminate\container\BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Object(Closure), Object(Closure))
#8 C:\LoginsApi\vendor\illuminate\container\Container.php(572): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Object(Closure), Array, NULL)
#9 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(291): Illuminate\Container\Container->call(Object(Closure), Array)
#10 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(265): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#11 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#12 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(418): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\LoginsApi\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\LoginsApi\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (InvalidArgumentException(code: 0): View [login] not found. at C:\\LoginsApi\\vendor\\illuminate\\view\\FileViewFinder.php:137)
[stacktrace]
#0 C:\\LoginsApi\\vendor\\illuminate\\view\\FileViewFinder.php(79): Illuminate\\View\\FileViewFinder->findInPaths('login', Array)
#1 C:\\LoginsApi\\vendor\\illuminate\\view\\Factory.php(130): Illuminate\\View\\FileViewFinder->find('login')
#2 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\helpers.php(431): Illuminate\\View\\Factory->make('login', Array, Array)
#3 C:\\LoginsApi\outes\\web.php(32): view('login')
#4 [internal function]: Closure->{closure}()
#5 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(29): call_user_func_array(Object(Closure), Array)
#6 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(75): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#7 C:\\LoginsApi\\vendor\\illuminate\\container\\BoundMethod.php(31): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Object(Closure), Object(Closure))
#8 C:\\LoginsApi\\vendor\\illuminate\\container\\Container.php(572): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Object(Closure), Array, NULL)
#9 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(291): Illuminate\\Container\\Container->call(Object(Closure), Array)
#10 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(265): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#11 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#12 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(418): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\LoginsApi\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\LoginsApi\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
